lib/Doctrine/Import/Builder.php

```diff
@@ -401,7 +401,7 @@ public function buildTableDefinition(array $definition)
      * @param  array $relations
      * @return string
      */
-    public function buildSetUp(array $definition)
+    public function buildSetUp(array &$definition)
```

Comment:
This is a BC break, as using reference changes the public interface and its behaviour.